controllers/helper.go

```diff
@@ -38,3 +41,23 @@ func GetImagePullSecrets() []string {
 		return []string{}
 	}
 }
+
+func formatJSON(str string) (string, error) {
+	var prettyJSON bytes.Buffer
+	if err := json.Indent(&prettyJSON, []byte(str), "", "    "); err != nil {
+		return "", err
+	}
+	return prettyJSON.String(), nil
+}
+
+func getLinkState(state string) string {
```

Comment:
why was this method moved ? as its only used in tests it should stay where it was IMO.

Reply:
fair, I will move back this back into the test files